internal/db/db.go

```diff
@@ -62,6 +63,15 @@ type BlueprintWithNoBody struct {
 	LastModifiedAt time.Time
 }
 
+type ServiceSnapshots struct {
+	Compliance *ComplianceSnapshot `json:"compliance,omitempty"`
+}
+
+type ComplianceSnapshot struct {
+	PolicyId        uuid.UUID `json:"policy_id"`
+	PolicyBlueprint string    `json:"policy_blueprint"`
```

Comment:
I wonder why we're storing the toml, and not the customizations directly (achieved by going through `lintOpenscap`, it also returns customizations.
```
	var cust Customizations
	_, err := h.lintOpenscap(ctx, &cust, true, distro, policy.String())
```

That way if the output we get from the compliance service ever changes, we won't need to keep support for converting all blueprint versions.

Reply:
good point @croissanne! This is definitely worth discussing. You're right that storing `Customizations` directly would have several technical advantages:
No TOML parsing overhead, 
Changes in compliance service output format won't break old snapshots
storing TOML:
Clear record of what was actually produced from each policy,
 Matches the blueprint API format users expect
 he original requirement specifically mentions "store the blueprint toml generated from a policy" - do we need the exact TOML output for: 
Compliance auditing?
Reproducing historical blueprint states?
WDYT? should we storing `Customizations` instead of TOML?